tests/test_text/test_dispersion.py

```diff
@@ -47,12 +52,10 @@ def test_quick_method(self):
 
         self.assert_images_similar(ax=ax, tol=25)
 
-    def test_integrated_dispersionplot(self):
+    def test_integrated_dispersion_plot(self):
```

Comment:
Thanks for resyncing the baseline images with this name change (and the below name changes) and cleaning up the older images!